string joint_name
string model_name
